### PR TITLE
Use different K8s version for capm3 0.5 tests

### PR DESCRIPTION
**Pull request by Rozzii**
As capm3 0.5 was getting older it started to have issues with newer
K8s versions thus it was decided to pin the K8s versions used in 0.5 tests.

### CODE REVIEW

File: tests/feature_tests/feature_test_vars.sh [hunk 1]
```diff
@@ -2,5 +2,10 @@
 
 export NUM_OF_CONTROLPLANE_REPLICAS="3"
 export NUM_OF_WORKER_REPLICAS="1"
-export FROM_K8S_VERSION="v1.24.0"
-export KUBERNETES_VERSION=${FROM_K8S_VERSION}
\ No newline at end of file
+if [ "${CAPM3RELEASEBRANCH}" == "release-0.5" ];
+then
+  export FROM_K8S_VERSION="v1.23.8"
```
Comment (furkatgofurov7) [inline tests/feature_tests/feature_test_vars.sh:7]:
```suggestion
  export FROM_K8S_VERSION="v1.23.5"
```

Comment (Rozzii) [reply tests/feature_tests/feature_test_vars.sh]: I have pushed the fix, page was just probably slow to update.

**Review by furkatgofurov7 — APPROVED** [review]
/approve

/cc @fmuyassarov @Sunnatillo

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Pull request merged**